guide/using-webots-makefiles.md

```diff
@@ -60,15 +60,16 @@ controller directory is "my\_controller" then the list must contain either
 
 ### Using the Compiler and Linker Flags (C/C++)
 
-These two variables can be used to pass flags to the gcc compiler or linker.
+These variables can be used to pass flags to the gcc compiler or linker.
 
 %figure "Webots Makefile Compiler Variables"
 
-| Variable  | Usage                                                                 |
-| --------- | --------------------------------------------------------------------- |
-| CFLAGS    | Specifies a list of flags that will be passed to the gcc/g++ compiler |
-| INCLUDE   | Specifies a list of flags that will be passed to the gcc/g++ compiler |
-| LIBRARIES | Specifies a list of flags that will be passed to the linker           |
+| Variable  | Usage                                                                                                      |
+| --------- | ---------------------------------------------------------------------------------------------------------- |
+| CFLAGS    | Specifies a list of flags that will be passed to the gcc/g++ compiler                                      |
+| LFLAGS    | Specifies a list of flags that will be passed to the linker                                                |
+| INCLUDE   | Specifies directories to be added to the list of places searched by the gcc/g++ compiler for include files |
+| LIBRARIES | Specifies directories and libraries to be added to the list of places searched by the linker               |
```

Comment:
I would rephrase the last one as: "Specifies library search directories and libraries passed to the linker"

Reply:
Ok, but this too is not very clear for me.